java/src/test/android/app/src/androidTest/java/ai/onnxruntime/example/javavalidator/SimpleTest.kt

```diff
@@ -54,14 +59,17 @@ class SimpleTest {
     }
 
     @Throws(OrtException::class, IOException::class)
-    fun runSigmoidModelTestImpl(intraOpNumThreads: Int, useNNAPI: Boolean = false) {
-        reportHelper.label("Start Running Test with intraOpNumThreads=$intraOpNumThreads, useNNAPI=$useNNAPI")
+    fun runSigmoidModelTestImpl(intraOpNumThreads: Int, useNNAPI: Boolean = false, useQNN: Boolean = false) {
```

Comment:
instead of adding another bool parameter, maybe specify the EP to use with an enum? I assume we don't need to test with more than one non-CPU EP at a time. perhaps there can be an [OrtProvider](https://github.com/microsoft/onnxruntime/blob/08cc2612f46484030ad92ddf0baac9ab3fae0f6c/java/src/main/java/ai/onnxruntime/OrtProvider.java#L11) parameter.